js/core/utils/date.js

```diff
@@ -609,8 +609,12 @@ var makeDate = function(date) {
 };
 
 var getDatesOfInterval = function(startDate, endDate, step) {
-    var currentDate = new Date(startDate.getTime()),
-        result = [];
+    let currentDate = new Date(startDate.getTime());
+    let result = [];
```

Comment:
`result` variable can be used as const, 
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Statements/const

Reply:
You're right that is can be `const`, but as for me, `result` should be set as `let` because of it's changing value throughout method execution (like in C-like languages with strict type management).